src/mod/build.ps1

```diff
@@ -90,7 +90,10 @@ function Update-Packages {
         $layoutFile | ConvertTo-Json | Out-File $layoutFilePath -Encoding ASCII
     
         Write-Host "Build finished."
-    }   
+    }
+
+    # Report success to the GitHub workflow
+    exit 0
```

Comment:
Without this the GitHub workflow thinks the script exited with code `1` and considers it a failure, halting the rest of the workflow.